### ISSUE TITLE
Do we want different categories of audience as part of tumpang tanya?

### ISSUE THREAD

**Issue by chungisad** [Sinar/tumpangtanya#28]
For example audiences such as: 

-if you're a student
-journalists
-CSO members

-what are each group of people usually interested in, and the thematic areas they really want to know 

I am not sure if this is already redundant with the development themes, SDGs, etc.

**Comment by kaerumy** [comment]
No. It shouldn't be a category. It's simply entry points to learning more and engaging with FOI.

eg.

Learning about making FOI requests:
 - For Journalists
 - For Students
 - For Civil Society
